packages/SwingSet/test/test-controller.js

```diff
@@ -35,6 +35,7 @@ test('load empty', async t => {
 
 async function simpleCall(t) {
   const config = {
+    defaultManagerType: 'local',
```

Comment:
Nit: Maybe define a `const managerType = 'local'` and use both here and in `stateFor` ?

Reply:
maybe, but if you were to edit that line and set it to `xsnap`, then the `workerOptions` in `stateFor()` would need to include a couple of lockdown/supervisor bundleIDs too, violating the expectation that the quick-to-modify-one-line setting would be implying